cpp/server/ct-dns-server.cc

```diff
@@ -22,7 +22,7 @@ DEFINE_string(domain, "", "Domain");
 DEFINE_string(db, "", "Database for certificate and tree storage");
 
 // Basic sanity checks on flag values.
-static bool ValidatePort(const char* flagname, int32_t port) {
+static bool ValidatePort(const char*, int32_t port) {
   if (port <= 0 || port > 65535) {
     std::cerr << "Port value " << port << " is invalid. " << std::endl;
```

Comment:
Or, could probably skip this line entirely, since the message that gflags give us is this (which is no less informative):

    ERROR: failed validation of new value '70000' for flag 'port'